pkg/registry/apis/folders/legacy_storage.go

```diff
@@ -67,16 +66,16 @@ func (s *legacyStorage) List(ctx context.Context, options *internalversion.ListO
 	}
 
 	parentUID := ""
-	// translate grafana.app/* label selectors into field requirements
-	requirements, newSelector, err := entity.ReadLabelSelectors(options.LabelSelector)
-	if err != nil {
-		return nil, err
-	}
-	if requirements.Folder != nil {
-		parentUID = *requirements.Folder
-	}
-	// Update the selector to remove the unneeded requirements
-	options.LabelSelector = newSelector
+	// // translate grafana.app/* label selectors into field requirements
+	// requirements, newSelector, err := entity.ReadLabelSelectors(options.LabelSelector)
+	// if err != nil {
+	// 	return nil, err
+	// }
+	// if requirements.Folder != nil {
+	// 	parentUID = *requirements.Folder
```

Comment:
We need to decide how to best specify this.  In entity store, we would mutate field labels -- we can continue with that approach, or switch to field selectors 🤷🏻